shared/src/main/scala/mouse/try.scala

```diff
@@ -15,4 +18,10 @@ final class TryOps[A](val ta: Try[A]) extends AnyVal {
     }
 
   @inline final def toEither: Either[Throwable, A] = cata[Either[Throwable, A]](Right(_), Left(_))
+
+  /**
+    * Converts a `Try[A]` to a `EitherT[F, Throwable, A]`.
+    */
+  def toEitherT[F[_]](implicit F: Applicative[F]): EitherT[F, Throwable, A] =
```

Comment:
For consistency with the surrounding mouse source code, please drop the doc comment (which essentially re-states the signature) and add an `@inline final`

Reply:
Shall do. Out of curiosity, besides consistency, wouldn't the `final` on the method be redundant as the class itself is already `final`?